selfdrive/car/subaru/values.py

```diff
@@ -240,6 +240,7 @@ class SubaruCarInfo(CarInfo):
       b'\xcb\"`@\a',
       b'\xcb\"`p\a',
       b'\xf1\x00\xa2\x10\n',
```

Comment:
Just a heads up, you're welcome to remove FW versions from other brand's queries for ECUS when adding new FW versions. So for example `b'\xf1\x00\xa2\x10\n',` comes from a Hyundai query. I'll do a big clean up of what I can soon